xcp_d/utils/modified_data.py

```diff
@@ -65,7 +65,7 @@ def interpolate_masked_data(bold_data, tmask, TR=1):
     flagged by the temporal mask.
 
     NOTE: TS- Why are slice times being inferred from the number of volumes?
-    Am I missing somehting?
+    Am I missing something?
```

Comment:
I think it's just bad commenting/naming - if there's something that makes more sense, we can definitely change it

Reply:
Hopefully we'll be able to just drop this whole step soon in favor of Nilearn.